simulation/traffic_simulator/src/entity/entity_base.cpp

```diff
@@ -72,8 +109,34 @@ void EntityBase::requestSpeedChange(
 {
   switch (transition) {
     case speed_change::Transition::LINEAR: {
-      setAccelerationLimit(std::fabs(constraint.value));
-      setDecelerationLimit(std::fabs(constraint.value));
+      job_list_.append(
+        /**
+           * @brief Checking if the entity reaches target speed.
+           */
+        [this, target_speed, constraint]() {
+          double diff =
+            target_speed.getAbsoluteValue(other_status_) - getStatus().action_status.twist.linear.x;
+          /**
+           * @brief Hard coded parameter, threashold for difference
+           */
+          if (std::fabs(diff) <= 0.1) {
```

Comment:
There is no advantage to explicitly using std::fabs, since std::abs defines overloading of floating-point types.

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/752/commits/5a98076c35f08817bb02f1fb575bacfcd0cd46d7